src/pages/scripts/getting-started/index.md

```diff
@@ -0,0 +1,81 @@
+<!-- 
+IDJS banner 
+Intro to scripts
+Prerequiste - read through Introduction -> Scripts
+-->
+
+
+# Building your first script
+
+This tutorial will walk you through the process of building your first script. You will learn how to create a script, and then run it in InDesign.
+
+## Prerequisites
+
+Before you start, make sure you have the following software installed on your machine:
+
+- **InDesign 2023 (v18.0)** or later (can be installed from the _Creative Cloud Desktop_ app)
+- **UXP Developer Tool v1.7** (can be [installed from the _Creative Cloud Desktop_ app](https://creativecloud.adobe.com/apps/download/uxp-developer-tools))
+- A **code editor** of your choice (for example, [Visual Studio Code](https://code.visualstudio.com/))
+
+## Steps
+
+### Step 1: Create a script
+
+Open your code editor and create a plain text file and add the following JavaScript code snippet to it.
+
+```js
+let { app } = require("indesign");
+let myDocument = app.documents.add();
+let myTextFrame = myDocument.pages.item(0).textFrames.add();
+
+// set the geometric bounds, the top, left, bottom, and right edges, of the text frame.
+myTextFrame.geometricBounds = ["6p", "6p", "24p", "24p"];
+myTextFrame.contents = "Hello World!";
+```
+
+Remember that an `idjs` file is a mere JavaScript file with a special extension to identify it as a UXP script for InDesign.
+
+### Step 2: Save the file to Scripts panel
+
+Scripts can only be executed once they appear in the Scripts panel. To add your scripts to the panel, you have two options
+1. Save your scripts to the InDesign application scripts folder.
+    macOS - `/Applications/Adobe InDesign 2023/Scripts`
+    Windows - //TODO path?
```

Comment:
@kareemmyk & @anoopvalomkot - What's the path for Windows?

Reply:
![image](https://github.com/AdobeDocs/uxp-indesign/assets/113042047/b3cf22ed-0cac-4d81-a5a4-c1cf8c42caae)
![image](https://github.com/AdobeDocs/uxp-indesign/assets/113042047/d9e6cbd1-db44-4b1f-9f9c-4421a34f6287)

@padmkris123 The user's folder path can be accessed by right clicking and selecting the 'reveal in finder' or 'reveal in explorer' options against user folder in scripts panel. This opens a folder under user's preferences. 

The path you have captured seems to be the install path, keeping scripts there might cause scripts to be deleted on app update/uninstall etc.

Reply:
@anoopvalomkot  I have taken the paths from the previously published docs.  Are you saying we should not suggest that to the developers?

Reply:
@padmkris123 Below is the path from Scripting guide documentation, I guess you are using same.

Windows: C:\Users\<username>\AppData\Roaming\Adobe\InDesign\Version 19.0\<locale>\Scripts
Mac: /Users/<username>/Library/Preferences/Adobe InDesign/Version 19.0/<locale>/Scripts

As long as the major version remains same, all of them point to same location. I will check internally and get back if the preferences get copied during installation of a different major version

Reply:
Preferences (and scripts within it) gets copied when updating InDesign if user chooses to, on the dialog from CCD.
![image](https://github.com/AdobeDocs/uxp-indesign/assets/113042047/67f1c9e2-5a09-46af-a90e-25e892a9c354)

Reply:
@padmkris123 
The Scripts path under application folder for InDesign in windows from my local setup is below:
C:\Program Files\Adobe\Adobe InDesign 2024\Scripts

But the catch here is that the path could be different on different versions of Windows. I am checking internally if install path is documented anywhere.